tmpfiles.d/eos-prune-printers.conf

```diff
@@ -0,0 +1,6 @@
+# remove stamp file from eos-config-printer
+# (from 4.0 use driverless printers)
+R /var/lib/eos-config-printer
+
+# clean up the residual print cache files
+R /var/cache/cups/*
```

Comment:
Wait, this will wipe the CUPS cache files on every boot. I think you want to only do this once in the script. Really both could probably be done in the script. `eos-config-printer` will never run again since it's not included in the OS anymore, so a one time cleanup is probably all that's needed.

Reply:
I make `tmpfiles.d/eos-prune-printers.conf ` installed in [`/usr/share/eos-boot-helper/tmpfiles.d/`](https://github.com/endlessm/eos-boot-helper/blob/895fe92e2967ff08d85ae106f4702d0c77b67f77/Makefile.am#L127) originally.  Will this be taken by systemd-tmpfiles on every boot?

> I think you want to only do this once in the script.
Do you suggest that just the simple `rm` to replace the `systemd-tmpfiles --remove ...` in the script?

Reply:
Okay!  After quick test, I believe the the simple `rm` works as well.